package.json

```diff
@@ -7,19 +7,19 @@
   "description": "Dat Desktop App",
   "author": "Dat Team <community@datproject.org>",
   "scripts": {
-    "start": "cross-env NODE_V=\"$(node -v)\" NODE_ENV=development electron .",
-    "pretest": "npm run lint",
-    "test": "npm run test:unit",
+    "start": "npm run build:dev && cross-env NODE_ENV=development npm-run-all -p start:watch start:electron",
```

Comment:
This script-command(prestart) is not recommended ?

`"prestart": "npm run build:dev",`
`"start": "cross-env NODE_ENV=development npm-run-all -p start:watch start:electron",`

Reply:
`prestart` should work: Do you want to make a new PR?